protocol-units/bridge/move-modules/sources/atomic_bridge_initiator.move

```diff
@@ -364,7 +367,7 @@ module atomic_bridge::atomic_bridge_initiator {
     }
 
     #[test(creator = @moveth, aptos_framework = @0x1, sender = @0xdaff, atomic_bridge = @atomic_bridge)]
-    #[expected_failure]
+    #[expected_failure ( abort_code = EWRONG_PREIMAGE )]
```

Comment:
Nothing major but you can also tighten this constraint with 
`#[expected_failure(abort _code = EWRONG_PREIMAGE, location = Self)]`

Probably can drop the spacing here